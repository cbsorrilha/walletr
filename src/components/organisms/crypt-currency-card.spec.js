import React from 'react';
import { render } from '@testing-library/react';
import CryptCurrencyCard from './crypt-currency-card';

describe('Test the crypt-currency-card component', () => {
  it('should match the snapshots', () => {
    const converter = jest.fn();
    const component = render(<CryptCurrencyCard value={0.0} coin="a" converter={converter} />);
    expect(component).toMatchSnapshot();
    expect(converter).toBeCalled();
    expect(converter).toBeCalledWith(0.0);
  });
});
