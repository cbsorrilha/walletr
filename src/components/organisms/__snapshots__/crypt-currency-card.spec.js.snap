// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the crypt-currency-card component should match the snapshots 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bwzfXH ioAeWq"
        direction="column"
      >
        <div
          class="sc-bwzfXH dqCzwa"
          height="auto"
        >
          <div
            class="sc-htpNat idAcKH"
            width="auto"
          >
            <p
              class="sc-bdVaJa iafxnE"
              color="#1cee6c"
            >
              a
            </p>
          </div>
          <div
            class="sc-htpNat kqCrsI"
          >
            <p
              class="sc-bdVaJa iafxnE"
              color="#1cee6c"
            >
              0
            </p>
          </div>
        </div>
        <div
          class="sc-bwzfXH hwDfLT"
          height="auto"
        >
          <div
            class="sc-htpNat idAcKH"
            width="auto"
          >
            <p
              class="sc-bdVaJa kVcfOV"
              color="#1cee6c"
            >
              R$
            </p>
          </div>
          <div
            class="sc-htpNat kqCrsI"
          >
            <p
              class="sc-bdVaJa AEPFh"
              color="#1cee6c"
            >
               NaN
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bwzfXH ioAeWq"
      direction="column"
    >
      <div
        class="sc-bwzfXH dqCzwa"
        height="auto"
      >
        <div
          class="sc-htpNat idAcKH"
          width="auto"
        >
          <p
            class="sc-bdVaJa iafxnE"
            color="#1cee6c"
          >
            a
          </p>
        </div>
        <div
          class="sc-htpNat kqCrsI"
        >
          <p
            class="sc-bdVaJa iafxnE"
            color="#1cee6c"
          >
            0
          </p>
        </div>
      </div>
      <div
        class="sc-bwzfXH hwDfLT"
        height="auto"
      >
        <div
          class="sc-htpNat idAcKH"
          width="auto"
        >
          <p
            class="sc-bdVaJa kVcfOV"
            color="#1cee6c"
          >
            R$
          </p>
        </div>
        <div
          class="sc-htpNat kqCrsI"
        >
          <p
            class="sc-bdVaJa AEPFh"
            color="#1cee6c"
          >
             NaN
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
